src/main/kotlin/no/skatteetaten/aurora/gobo/security/currentUser.kt

```diff
@@ -16,6 +19,10 @@ const val GUEST_USER_ID = "anonymous"
 const val GUEST_USER_NAME = "Gjestebruker"
 val ANONYMOUS_USER = User(GUEST_USER_ID, GUEST_USER_NAME)
 
+fun <T> runBlockingWithTimeout(block: suspend CoroutineScope.() -> T): T {
+    return runBlocking { withTimeout(5000) { block() } }
```

Comment:
Jeg har ikke vært med på diskusjonen for akkurat dette. Hvorfor er 5 sekunder valgt som timeout her? Magefølelsen sier at det kan være litt kort tid(?)

Reply:
Jeg tror nesten vi kan sette ned den tiden men har latt den stå sånn foreløpig. I et normal tilfelle så vil det allerede vært kjørt en TokenReview. Resultatet av det vil caches en liten periode, og det er cachen vi vil treffe her.